temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivitySlotSupplierQueue.java

```diff
@@ -95,9 +97,22 @@ private void processQueue() {
         }
         request.task.getExecutionContext().setPermit(slotPermit);
         afterReservedCallback.apply(request.task);
+      } catch (InterruptedException e) {
+        Thread.currentThread().interrupt();
+      } catch (Throwable e) {
+        // Fail the workflow task if something went wrong executing the local activity (at the
+        // executor level, otherwise, the LA handler itself should be handling errors)
+        log.error("Unexpected error executing local activity", e);
```

Comment:
Can we make this error message more clear? Without reading the comment it sounds like the error could be from the users activity.
```suggestion
        log.error("Unexpected error submitting local activity task to worker", e);
```
Does this seem accurate?

Reply:
Yeah, I thought for a bit about what to write here. I had "scheduling" but that felt unclear too. Your wording works for me.